<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KarelReplaceUsagesDialog</class>
 <widget class="QDialog" name="KarelReplaceUsagesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Replace Usages</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="replaceButton">
       <property name="text">
        <string>Replace</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>KarelReplaceUsagesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
